app/src/main/java/com/yeonae/chamelezone/view/mypage/MoreButtonFragment.kt

```diff
@@ -53,15 +58,14 @@ class MoreButtonFragment : BottomSheetDialogFragment() {
 
             targetFragment?.onActivityResult(targetRequestCode, BTN_DELETE, data)
 
-            activity.data(myReviewData)
+            deletedButtonListener.onDeleteSelected(myReviewData)
 
             dismiss()
         }
     }
-
     override fun onAttach(context: Context) {
         super.onAttach(context)
-            activity = context as MyReviewActivity
+        deletedButtonListener = (context as OnDeletedSelectedListener)
```

Comment:
```suggestion
        (context as OnDeletedSelectedListener)?.let{
                    deletedButtonListener = it
        }
```
해당 fragment의 부모 activity에서 OnDeletedSelectedListener를 implements하고 있지 않을경우,
as 시에 에러가 발생합니다. 이런식으로 예외사항 처리를 하면 좋을 것 같아요.

Reply:
수정했습니다
11daf54